README.md

```diff
@@ -0,0 +1,97 @@
+# appu
+
+Appu is a toolkit for podcast edition and publishing.
+
+## Rationale
+
+While running the [Entre Dev Y Ops podcast](https://www.entredevyops.es), the authors found interesting to start building a set of tools to make this easier. We hope this might help anyone else.
+
+Currently we start preparing every episode by writing a simple script we store in a shared Drive folder.
+
+We record it using some videoconferencing tool, then we convert the video file to an audio track we store.
+
+We also have an S3 bucket and CDN configured to serve the audio files and the RSS feed. After we published the audio file, we also publish a new article about it in our blog.
+
+Finally, using the tools here, and our own podcast configuration file, we have most of this automated, so the mistakes are the less and we can publish sooner.
+
+## Usage
+
+### Requirements
+
+You'll need the following:
+- Docker, for Windows and Mac, you should use Docker Desktop.
+- Publishing infrastructure based on an online storage, and a CDN with invalidation features.
+  Currently only AWS S3 and CloudFront are supported.
+  You'll need credentials for uploading the episodes' audio files and the RSS feed and then invalidating them on the CDN.
```

Comment:
You'll need the following:
- Docker, for Windows and Mac, you should use Docker Desktop.
- Publishing infrastructure based on an online storage, and a CDN with invalidation features (currently only AWS S3 and CloudFront are supported).
- Valid credentials for uploading the episodes' audio files and the RSS feed, and then invalidating them on the CDN.